test/index.js

```diff
@@ -144,6 +144,25 @@ var IdentityArb = function(arb) {
   return arb.smap(Identity, function(i) { return i.value; });
 };
 
+var runCompositionTests = function(compose){
+
+  it('is a ternary function', function() {
+    eq(typeof compose, 'function');
+    eq(compose.length, 3);
+  });
+
+  it('composes two functions assumed to be unary', function() {
+    eq(compose(R.map(Math.sqrt), JSON.parse, '[1, 4, 9]'), [1, 2, 3]);
+  });
+
+  it('is curried', function() {
+    eq(compose(R.map(Math.sqrt)).length, 2);
+    eq(compose(R.map(Math.sqrt))(JSON.parse).length, 1);
+    eq(compose(R.map(Math.sqrt))(JSON.parse)('[1, 4, 9]'), [1, 2, 3]);
+  });
+
+};
+
```

Comment:
This is here because I didn't want to repeat existing code, yet I wanted both functions (`S.compose` and `S.B`) to be tested, because they are in fact different functions, despite being aliases.